src/styles/Components/src/tokens-card.css

```diff
@@ -0,0 +1,56 @@
+:root {
+  --denhaag-card__actions-padding-block-end: 0;
+  --denhaag-card__actions-padding-block-start: 8px;
+  --denhaag-card__actions-padding-inline-end: 0;
+  --denhaag-card__actions-padding-inline-start: 0;
```

Comment:
You're using `__` in the design tokens here, this should be replaced by a `-`, so for example: `--denhaag-card-actions-padding-block-end`

Reply:
Resolved